pom.xml

```diff
@@ -107,6 +107,11 @@
                     </exclusion>
                 </exclusions>
             </dependency>
+            <dependency>
+                <groupId>org.apache.zookeeper</groupId>
+                <artifactId>zookeeper</artifactId>
+                <version>3.7.2</version>
```

Comment:
Is there a version of the zookeeper version number available from the common pom.xml?

Reply:
unfortunately, the version from common is [3.8.3](https://github.com/confluentinc/common/blob/67e880a6a62c5a5f3bf5467c05ec408c77b28172/pom.xml#L99), which is causing the issue, I'll add a comment to this.